### PR TITLE
Add support for creating multiple droplets.

### PR DESCRIPTION
**Pull request by iheanyi**
Allows godotto to create multiple droplets and have them be spied on accordingly.

### CODE REVIEW

File: pkg/droplets/droplets.go [hunk 1]
```diff
@@ -111,3 +112,28 @@ func (svc *dropletSvc) list(all otto.FunctionCall) otto.Value {
...
+	}
+
+	var d = make([]otto.Value, 0)
+	for _, droplet := range droplets {
+		d = append(d, godojs.DropletToVM(vm, droplet.Struct()))
+	}
+
+	v, err := vm.ToValue(d)
+	if err != nil {
+		ottoutil.Throw(vm, err.Error())
```
Comment (table-delete) [inline pkg/droplets/droplets.go]: What's going to happen to the created droplets here?  Do we need to clean up?

File: pkg/droplets/droplets_test.go [hunk 2]
```diff
@@ -20,6 +20,7 @@ assert(pkg != null, "package should be loaded");
 assert(pkg.list != null, "list function should be defined");
 assert(pkg.get != null, "get function should be defined");
 assert(pkg.create != null, "create function should be defined");
+assert(pkg.create_multiple != null, "create_multiple function should be defined");
```
Comment (table-delete) [inline pkg/droplets/droplets_test.go]: There's a test for Create() below... not going to add a test for CreateMultiple?

File: pkg/droplets/droplets.go (hunk 1 above)
Comment (iheanyi) [reply pkg/droplets/droplets.go]: @table-delete Not in this project. In canary, they'll get automagically deleted by the garbage collector.

File: pkg/droplets/droplets_test.go (hunk 2 above)
Comment (iheanyi) [reply pkg/droplets/droplets_test.go]: Good call, thanks!

File: pkg/extra/do/cloud/droplets/client.go [hunk 3]
```diff
@@ -70,6 +89,29 @@ func (svc *client) Create(ctx context.Context, name, region, size, image string,
...
+	opt.req.Size = size
+	opt.req.Region = region
+	opt.req.Image.Slug = image
+
+	r, resp, err := svc.g.Droplets.CreateMultiple(ctx, opt.req)
+	if err != nil {
+		return nil, err
+	}
+
+	droplets := make([]Droplet, 0)
```
Comment (aybabtme) [inline pkg/extra/do/cloud/droplets/client.go]:
```go
droplets := make([]Droplet, 0, len(r))
```

**Comment by aybabtme** [comment]
:shipit: after adding the test that @table-delete mentionned

**Comment by iheanyi** [comment]
Waiting on CI to pass before merging

**Pull request merged**